### DISCUSSION TITLE
参加希望

### DISCUSSION THREAD

**Discussion by yamashita-kenngo** [Personal-Data-Linkage-Module/pxr-ver-1.0#104] [メンバー加入希望]
広島県廿日市市でシビックテック活動をしている山下と申します。こちらのパーソナルデータ連携モジュールの運営・開発に参加を希望します。

**Comment by r-saito-cw** [comment]
@yamashita-kenngo 、初めまして！本コミュニティの運営側として参加している齊藤と申します。山下さんのアカウントをメンバとして登録しようと思います。また、Slackなどコミュニケーションの場についてもご案内しますね。お待ちください！

**Reply by r-saito-cw** [reply]
inviteを送信しました。
また、OSSコミュニティではCode For Japan様で運営しているSlackチャンネルの一つをコミュニケーションの場としています。
メンバは、隔週でZoomによる定例会議を開いており、モジュールに関するメンテナンスの状況や、貢献協力依頼のほか、DSAの活動についても情報共有をしています。

こちらにてCfJさんのSlackコミュニティに加入いただければ、活動場所であるチャンネルの内容をご覧いただけます。
https://www.code4japan.org/activity/community
チャンネル
#proj-cityos_auth

**Reply by r-saito-cw** [reply]
@yamashita-kenngo  当日の案内となってしまいましたが、定期的に行っているコミュニティのZoom会合が本日14時から開かれます。毎月２回、木曜の14時から開催しています。もしご参加を希望されるようでしたら個別にZoomのIDなどお送りいたしますので、おっしゃってください。

**Comment by r-saito-cw** [comment]
メンバ追加処理が完了しましたので、いったん本ディスカッションはCLOSEします。
